catalog/imaging/catalog-imaging-transformer/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+ -->
+<blueprint xmlns="http://www.osgi.org/xmlns/blueprint/v1.0.0">
+
+    <bean id="metacardType" class="ddf.catalog.transformer.nitf.NitfMetacardType"/>
+
+    <bean id="transformer" class="ddf.catalog.transformer.nitf.NitfInputTransformer">
+        <property name="nitfMetacardType" ref="metacardType"/>
+    </bean>
+
+    <service ref="transformer" interface="ddf.catalog.transform.InputTransformer">
+        <service-properties>
+            <entry key="id" value="nitf"/>
+            <!-- shortname only exists for backwards compatibility -->
+            <entry key="shortname" value="nitf"/>
+            <entry key="title" value="NITF Input Transformer"/>
+            <entry key="description"
+                   value="Extracts metadata from detected NITF files"/>
+            <entry key="mime-type">
+                <list>
+                    <value>image/nitf</value>
+                </list>
+            </entry>
+        </service-properties>
+    </service>
+
+    <service ref="metacardType" interface="ddf.catalog.data.MetacardType">
+        <service-properties>
+            <entry key="id" value="nitfMetacardType"/>
```

Comment:
I think we came to a consensus today to make the key `name` and it should match what nitfMetacardType.getName().